packages/js/product-editor/src/blocks/generic/section/editor.scss

```diff
@@ -55,5 +55,8 @@
 	&__heading-description {
 		margin: 0;
 		max-width: 300px;
+		a {
+			text-decoration: none;
```

Comment:
why `none`?

<img width="208" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/48f50256-03b6-4c2b-a626-d65d9cf099cb">

Reply:
Thank you Jarek, for clarifying. I restored the underline in the commit https://github.com/woocommerce/woocommerce/pull/41844/commits/c3f5acc8eb2904be3e29d4ee41bd287117ec7cb2